technical-reports/format/design-token.md

```diff
@@ -62,7 +62,7 @@ Due to the syntax used for [token aliases](#aliases-references) the following ch
 
 ### Token value type
 
-Token values may be any valid JSON type:
+Token values MUST be a valid JSON type:
```

Comment:
I agree, and the statement "Token values MUST be a valid JSON type" is inaccurate since we're defining a number of additional types beyond the standard JSON set.